redash/query_runner/dynamodb_sql.py

```diff
@@ -0,0 +1,129 @@
+import json
+import logging
+import sys
+
+from redash.query_runner import *
+from redash.utils import JSONEncoder
+
+logger = logging.getLogger(__name__)
+
+try:
+    from dql import Engine, FragmentEngine
+    enabled = True
+except ImportError, e:
+    enabled = False
+
+types_map = {
+    'UNICODE': TYPE_INTEGER,
+    'TINYINT': TYPE_INTEGER,
+    'SMALLINT': TYPE_INTEGER,
+    'INT': TYPE_INTEGER,
+    'DOUBLE': TYPE_FLOAT,
+    'DECIMAL': TYPE_FLOAT,
+    'FLOAT': TYPE_FLOAT,
+    'REAL': TYPE_FLOAT,
+    'BOOLEAN': TYPE_BOOLEAN,
+    'TIMESTAMP': TYPE_DATETIME,
+    'DATE': TYPE_DATETIME,
+    'CHAR': TYPE_STRING,
+    'STRING': TYPE_STRING,
+    'VARCHAR': TYPE_STRING
+}
+
+
+class DynamoDBSQL(BaseSQLQueryRunner):
+    @classmethod
+    def configuration_schema(cls):
+        return {
+            "type": "object",
+            "properties": {
+                "region": {
+                    "type": "string",
+                    "default": "us-west-1"
+                },
+                "host": {
+                    "type": "string",
+                    "default": "127.0.0.1"
+                },
+                "port": {
+                    "type": "number",
+                    "default": 8000
+                },
+                "access_key": {
+                    "type": "string",
+                    "default": "anything"
+
+                },
+                "secret_key": {
+                    "type": "string",
+                    "default": "anything"
+
+                },
+                "is_secure": {
+                    "type": "boolean",
+                    "default": False,
+                }
+            },
+            "required": ["host"]
```

Comment:
Please add a `secret` field with array of the secrets (in this case `SECRET_KEY`).